package.json

```diff
@@ -217,6 +245,10 @@
                 "command": "vscode-fracas.recompileFracasObject",
                 "title": "Fracas: Recompile Previous Object"
             },
+            {
+                "command": "vscode-fracas.makeStringTableImport",
+                "title": "Fracas: Make String Table CPP"
+            },
```

Comment:
`TdpLocalization.cpp` is automatically generated when files are saved/created/deleted, but you can also invoke it manually using the `Fracas: Make String Table CPP` command.